snuba/clusters/cluster.py

```diff
@@ -274,6 +273,9 @@ def get_distributed_nodes(self) -> Sequence[ClickhouseNode]:
         ), "distributed_cluster_name must be set"
         return self.__get_cluster_nodes(self.__distributed_cluster_name)
 
+    def connection_tuple(self) -> Tuple[str, int]:
+        return (self.__query_node.host_name, self.__http_port)
```

Comment:
This is not the right port. We use the TCP one for queries. 
If you want to return the connection tuple from the cluster you should have two methods, one for TCP and one for HTTP

Reply:
Makes sense, and we don't care about the HTTP port, or should that be part of the connection info tuple?